libs/sdk-py/langgraph_sdk/client.py

```diff
@@ -239,15 +239,57 @@ def __init__(self, http: HttpClient) -> None:
         self.http = http
 
     async def get(self, assistant_id: str) -> Assistant:
-        """Get an assistant by ID."""
+        """Get an assistant by ID.
+
+        Args:
+            assistant_id (str): The ID of the assistant to get.
+
+        Returns:
+            Assistant: The returned assistant.
+
+        Example Usage:
+
+            assistant = await client.assistants.get(
+                assistant_id="my_assistant_id"
+            )
+
+        """  # noqa: E501
         return await self.http.get(f"/assistants/{assistant_id}")
 
     async def get_graph(self, assistant_id: str) -> dict[str, list[dict[str, Any]]]:
-        """Get the graph of an assistant by ID."""
+        """Get the graph of an assistant by ID.
+
+        Args:
+            assistant_id (str): The ID of the assistant to get the graph of.
+
+        Returns:
+            dict[str, list[dict[str, Any]]]: The graph information for the assistant.
```

Comment:
Would be good to include what that information actually is. Would be nice to type it as a TypedDict or something too.